### PR TITLE
Add Elixir integration

### PR DESCRIPTION
**Pull request by jayzes**

### CODE REVIEW

File: dots/bashrc [hunk 1]
```diff
@@ -74,16 +75,16 @@ parse_git_branch() {
   __git_ps1 " [%s]"
 }
 
-parse_ruby_version() {
-  echo " [$(rbenv version-name)]"
+parse_versions() {
+  echo " [rb:$(rbenv version-name)|ex:$(exenv version-name)]"
```
Comment (jayzes) [inline dots/bashrc:79]: Not sure if there's a nicer way to display this - it's not super relevant outside of a project in that language, but detecting is a little bit of a pain

**Comment by jayzes** [comment]
@mkitt this depends on a few Homebrew packages (`exenv` and `elixir-build`) - not sure the best place to document that?

**Comment by Justin-Holmes** [comment]
👍

**Comment by mkitt** [comment]
Cool! The Readme would work for documenting it. 👍

**Comment by jayzes** [comment]
Excellent, that's done. Going to go ahead and merge this one.

**Pull request merged**